src/Symfony/Bundle/FrameworkBundle/Resources/config/psr_http_message_bridge.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Psr\Http\Message\ResponseFactoryInterface;
+use Psr\Http\Message\ServerRequestFactoryInterface;
+use Psr\Http\Message\StreamFactoryInterface;
+use Psr\Http\Message\UploadedFileFactoryInterface;
+use Symfony\Bridge\PsrHttpMessage\ArgumentValueResolver\PsrServerRequestResolver;
+use Symfony\Bridge\PsrHttpMessage\EventListener\PsrResponseListener;
+use Symfony\Bridge\PsrHttpMessage\Factory\HttpFoundationFactory;
+use Symfony\Bridge\PsrHttpMessage\Factory\PsrHttpFactory;
+use Symfony\Bridge\PsrHttpMessage\HttpFoundationFactoryInterface;
+use Symfony\Bridge\PsrHttpMessage\HttpMessageFactoryInterface;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('psr_http_message_bridge.http_foundation_factory', HttpFoundationFactory::class)
+        ->alias(HttpFoundationFactoryInterface::class, 'psr_http_message_bridge.http_foundation_factory')
+
+        ->set('psr_http_message_bridge.psr_http_factory', PsrHttpFactory::class)
+            ->args([
+                service(ServerRequestFactoryInterface::class)->nullOnInvalid(),
+                service(StreamFactoryInterface::class)->nullOnInvalid(),
+                service(UploadedFileFactoryInterface::class)->nullOnInvalid(),
+                service(ResponseFactoryInterface::class)->nullOnInvalid(),
+            ])
+        ->alias(HttpMessageFactoryInterface::class, 'psr_http_message_bridge.psr_http_factory')
+
+        ->set('psr_http_message_bridge.psr_server_request_resolver', PsrServerRequestResolver::class)
+            ->args([service('psr_http_message_bridge.psr_http_factory')])
+            ->tag('controller.argument_value_resolver', ['priority' => -100])
```

Comment:
I'm trying to run this PR to see how it goes. I did install a webapp-pack and added the psr bridge.
Then I created a controller and added a PSR RequestInterface as argument.
Because this priority is below `ServiceValueResolver` , I've an error telling me now such service.
Then I changed the -100 to 50 (same level as RequestValueResolver), and the error is now "PSR-17 factories have been provided."
This is consistent with the fact that the bridge is ... a bridge between Symfony and ... a library that needs to be installed.
A solution to this could be to require `php-http/discovery` + `psr/http-factory-implementation`. That's the only way for us to provide a nice out of the box experience.

- or - 
we keep the pack + its recipe?